frontend/cypress/e2e/firearm-identification.cy.js

```diff
@@ -0,0 +1,63 @@
+describe('Firearm Identification', () => {
+  it('should identificate real firearm', () => {
+    cy.visit('/accueil')
+    cy.getByDataTestid('identification')
+      .contains('J’ai déjà mis mon arme en sécurité, je veux l’identifier')
+      .click()
+    cy.url()
+      .should('contain', '/instructions')
+    cy.url().should('contain', '/instructions')
+    cy.contains('h3', 'Pour un résultat optimal')
+    cy.contains('span', 'canon vers la droite')
+
+    cy.getByDataTestid('select-file').as('fileInput')
+    cy.intercept('POST', '/api/upload').as('upload')
+    cy.get('@fileInput').selectFile('./cypress/images/pistolet-semi-auto.jpg', { force: true })
+    cy.wait('@upload').then(({ response }) => {
+      expect(response.statusCode).to.eq(200)
+    })
+    cy.getByDataTestid('go-on-and-follow').click()
```

Comment:
Je pense que tu peux renommer partout le data-testid `go-on-and-follow` par `next-step` ça sera plus clair pour le test

Reply:
Correction faite